pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -32,6 +34,74 @@ var info = capabilities.MustNewCapabilityInfo(
 	"OCR3 consensus exposed as a capability.",
 )
 
+type Output = any
+
+type CapabilityConfig struct {
+	AggregationMethod string `json:"aggregationMethod" jsonschema:"required"`
+	AggregationConfig any    `json:"aggregationConfig" jsonschema:"required"`
+	Encoder           string `json:"encoder" jsonschema:"required"`
+	EncoderConfig     any    `json:"encoderConfig" jsonschema:"required"`
+	ReportID          string `json:"reportId" jsonschema:"required"`
+}
+
+type CapabilityInputs struct {
+	// This Input is coupled to the Datastreams Trigger. This was an
+	// intentional short-term trade-off that we'll fix in the future.
+	Observations workflows.CapabilityDefinition[[]datastreams.FeedReport]
+}
```

Comment:
I like this a lot because there's no need for merge steps anymore, it's free and more native feeling!  Good work!